### DISCUSSION TITLE
Abrir conta em um banco nos EUA

### DISCUSSION THREAD

**Discussion by bergergit** [DyegoCosta/trabalhando-remoto#185] [General]
Salve pessoal.

Após ler várias situações apresentadas pelo trabalhadores remotos daqui, me ocorreu que podemos ter uma nova possibilidade, que seria a abertura de uma conta bancaria nos EUA.

### Por que?
Muitos daqui já recebem justamente de uma empresa dos EUA (ou Canadá). A maioria recebe numa conta digital (PayPal, Payoneer, Bitwage) e ja transfere 100% do valor para a conta do Brasil, ou ja recebem diretamente na conta via remessa internacional. Em qualquer caso, sempre há um pouco de perda (taxa da conta digital, spread do banco, IOF).
Se abrirmos uma conta em um banco nos EUA, podemos transferir todo ou uma parte do dinheiro, para essa conta internacional. Seria uma transferencia direta simples, sem conversoes de dólares e taxas.

### Pra quem serviria?
Só funcionaria para quem não está usando 100% dos ganhos no Brasil. Ou seja, para quem ja está recebendo mais do que suas despesas, ou para quem está fazendo uns trabalhos extras, e não precisa transferir toda a grana para o Brasil por enquanto.

### É difícil abrir conta la?
Acho que não. Sei que existe o BB Americas (https://www.bbamericas.com/br/internacional/) mas tem vários bancos no Brasil, que possuem agencias no exterior.

### Como funcionaria?
Penso que a forma mais simples, seria receber o dinheiro do seu pagador em uma conta digital (tipo Bitwage), e dessa conta, transferir uma parte para o Brasil (para suas despesas), e uma outra parte, para o banco internacional

### Qual a vantagem?
Penso em algumas:

1. Menos taxas: Nao pagamos taxa de conversao de dólar, spread, etc, enquanto o dinheiro estiver la

1. Ao fazer uma viagem para os EUA por exemplo, poderíamos ter um cartão de crédito do país! Nesse caso estaríamos usando os dólares que ganhamos, sem ter que converter para dólar turismo, e sem pagar IOF!

1. Você protege seu dinheiro contra medidas economicas Brasileiras abusivas (lembram do plano Collor? :( )

1. Com o dinheiro la, fica mais facil enviar para uma corretora americana, e fazer investimentos nos Estados Unidos (investimentos muito mais sólidos do que aqui)

### Perguntas:

Para melhor organizar o tópico, vão aqui algumas perguntas minhas.

1. Será que essa é uma ideia viável?
1. Qual seria a melhor forma de receber este valor, do seu pagador? 
1. Como será que fica a declaração de impostos sobre essas transferencias? Só declaramos o valor que transferimos para o Brasil? Será que tem que declarar algo la nos EUA tambem?

Diante dessas ideias, gostaria muito de ouvir a opinião de vocês, se alguém tem experiencia com esse tipo de coisa, etc.

**Comment by samiraguiar** [comment]
Vou colocar aqui o que eu sei/acho, qualquer coisa alguém corrige.

1. Eu acho que sim, pra quem se encaixa nas condições que você mencionou. Se for para manter o dinheiro lá para uma viagem, por exemplo eu acho bem interessante. Inclusive estou fazendo isso com a Payoneer: vou para o exterior no final do ano e fui deixando uma parte dos meus ganhos na conta que tenho com eles. Como eles me dão um cartão pré-pago, posso usar para sacar em caixas eletrônicos no exterior (eles cobram $3.15/€2.50 por saque) e assim evito levar muito dinheiro ou pagar IOF e taxas de cartões como o Visa Travel Money.
Quanto a investir lá fora, já é mais complicado. As corretoras de lá costumam pedir um aporte inicial que varia de US$ 2 mil (Colmex) até US$ 25 mil, muito embora tenha algumas como a Tastyworks que não pedem aporte. É bem interessante pra quem quer fazer day trade, porque a bolsa americana tem muito mais movimentações ao dia. Outra ideia é usar como reserva financeira para caso a economia afunde como você disse (atualmente o congelamento de poupança como o Collor fez é vedado pela constituição). De qualquer forma, acho que as duas coisas são mais indicadas pra quem tem uma boa grana disponível (infelizmente não é meu caso).

2. Creio que nem precise pedir para o pagador depositar na conta do Bitwage, é só depositar na conta lá fora diretamente. Assim você mesmo manda a parte que vai utilizar no Brasil para o Bitwage, minimizando custos de transferência. Não sei o Bitwage, mas normalmente esses serviços não cobram de quem paga, mas cobram quando você vai enviar para outra conta (spread de conversão ou taxas fixas no caso de ser na mesma moeda).

3. Você precisa declarar todo montante que receber e que tiver no exterior e pagar imposto nesses proventos da mesma forma que faz aqui. Deve ter algumas peculiaridades no IR, mas isso eu já não sei bem.

P.S.: Eu dei uma lida rápida aqui e acho que precisa de US$ 10 mil iniciais quando a abertura é feita pela internet (e não presencial) no BB Americas.

**Comment by cirojr** [comment]
@samiraguiar, e vale a pena esse cartão da Payonner? Quase não há divulgação dele. Tem taxas? Você precisa receber valores com frequência pra ser elegível?
Tive a conta no LeoPay recém aberta já cancelada. A alegria durou bem pouco.

**Comment by bergergit** [comment]
Obrigado pelas infos @samiraguiar 

E agora me ocorreu o seguinte: Eu pretendo declarar imposto sobre os recebimentos vindos do Payoneer / Banco internacional, emitindo uma Nota Fiscal pela minha empresa (ME Simples)
Entao nesse caso, nao entendo como fica a declaração de impostos dos dólares no banco internacional. Pois se eu resolver transferir uma quantia para o Brasil, teria que pagar imposto duas vezes?

Uma outra pergunta: sabe dizer se Bitwage tambem fornece cartao de crédito? Li por aqui que ele é uma opçao melhor que Payoneer, mas nao achei dados sobre eles terem cartao.

Valeu

**Comment by samiraguiar** [comment]
@cirojr Infelizmente o cartão não é gratuito, ele tem uma anuidade de $29.95. Pegando a cotação do dólar turismo hoje, acima de uns 600 dólares já compensaria usar o cartão ao invés de comprar a moeda. Compras online ou locais no mesmo local de emissão do cartão também são gratuitas, e embora eu não tenha testado ainda, imagino que dê pra fugir do IOF também nas compras de passeios, estadia e transporte feitas no Brasil antes de viajar. Se você quiser ver todas as taxas, dá uma olhada [aqui](https://www.payoneer.com/fees/) na parte "OPTIONAL: Use the Payoneer Prepaid MasterCard" e [aqui](https://www.payoneerhow.com/payoneer-fees/).

Não precisa receber valores com frequência, com menos de 100 USD na conta, se não me engano, você já consegue pedir. Ele é pré-pago, então o saldo vai variar de acordo com o que você tiver na conta. Se não tiver na hora que forem cobrar anuidade, ele deixa pra cobrar quando você colocar dinheiro. Não tenho certeza, mas acho que só cobram depois de um ano e não logo que você ativa.

Quanto à Payoneer em si, já tive alguns problemas com conta jurídica que deram um trabalhinho pra resolver. Consegui resolver depois de causar bastante e falar com o pessoal da ouvidoria.

@bergergit Depende do imposto que você estiver falando. Até onde eu sei, você precisa emitir a NF sim, com [o valor de compra da moeda no dia anterior à emissão](http://www.efficienza.com.br/emissao-de-nota-fiscal/). O que eu faço é: recebo o dinheiro na Payoneer (vamos supor 1000 dólares). Emito a nota com o valor de compra do dólar no dia anterior de acordo com a [cotação do BC](https://www4.bcb.gov.br/pec/taxas/port/ptaxnpesq.asp?id=txcotacao), usando os 1000 dólares como valor bruto. Depois mando pra minha conta no Brasil e aí a Payoneer cobra as taxas dela (o spread). Não pago mais imposto relacionado a isso depois dessa transferência, então imagino que no seu caso seria igual.

Quanto ao Bitwage, eu não sei dizer se ele fornece cartão porque não cheguei a usar. Eu ia abrir conta, mas as taxas sempre acabavam ficando acima das da Payoneer. Mas pelo que falam aqui, o serviço deles parece ser bem melhor.

**Comment by bergergit** [comment]
Beleza, valeu pelas infos @cirojr 
Agora ficou mais claro: devemos declarar o imposto assim que emitirmos a invoice para o pagador, e considerando o valor total da invoice, usando a cotação do BC do dia anterior.
Eu achei erroneamente que só deveria declarar o montante que estaríamos transferindo para a conta do Brasil.

E para dar mais continuidade a esse assunto, eu andei fazendo umas contas, e cheguei à conclusão de que pode ser vantajoso mesmo abrir uma conta no BB Americas. **Mesmo para quem vai transferir 100% do valor para o Brasil.**

Isso porque na transferencia do BB Americas para uma conta sua no banco do Brasil, nao há cobranças de taxas, e eles usam o dólar comercial do dia como cotação! Só incide mesmo os 0,38% de IOF. 
Pelo que andei pesquisando, nenhum outro método de pagamento teria uma taxa tão baixa.
A ideia é pedir então para o seu pagador te pagar diretamente nessa sua conta bancária internacional.

O inconveniente do BB Americas é: Para abrir uma conta la, remotamente (sem ir na agencia), é preciso ter uma conta no Banco do Brasil, e transferir U$ 10.000,00 pra conta de la.
Voce pode pegar o valor de volta depois, mas se tiver menos que isso na conta la, passa a pagar U$ 15,00 mensais. Ainda assim, a taxa para transferir pro Brasil é menor do que o resto.
Ou, se eventualmente estiver viajando nos EUA, pode abrir diretamente numa agencia la. Aí, precisa de um depósito de U$ 3.000,00

Se eu deixei de incluir alguma conta aí, me corrijam por favor.

**Comment by samiraguiar** [comment]
@bergergit Engraçado, achei informações conflitantes no próprio site deles. [Nesse link](https://www.bbamericas.com/br/internacional/bb-remessa/) fala que não existem tarifas (na verdade, na seção de Perguntas Frequentes ele fala de uma tarifa de US$ 4.99 que no momento não está sendo cobrada).

Já em outra parte do site deles tem uma [lista de tarifas](https://www.bbamericas.com/br/tabela-de-tarifas-pessoa-fisica/) pra pessoa física e [outra](https://www.bbamericas.com/br/empresas/tabela-de-tarifas/) pra pessoa jurídica.
Tem dois itens ali que chamaram atenção: US$ 15 para recebimento em transferências domésticas (a empresa americana transferindo o dinheiro para a gente) e US$ 20 para envio para conta do BB (nós enviando para a conta no Brasil).

**Comment by bergergit** [comment]
Realmente, as informações do site deles são um pouco desencontradas.
Mas acabei de confirmar com um casal que está usando o serviço, que realmente, até hoje (Outubro/2018) eles não sofreram nenhuma cobrança. Apenas 1,1% de IOF.
_(Sim, teve uma mudança recente na tributação do IOF para transferencias entre mesma titularidade, aumentando o valor de 0,38% para 1,1%)._

Eles ouviram falar que talvez passem a fazer essa cobrança de U$ 20,00. 
Mas se cobrassem hoje em dia, caberia até uma bela discussão na ouvidoria, uma vez que boa parte do site ainda cita as "transferencias sem taxas".

Mas enfim... mesmo com a taxa, ainda sim seria menor do que qualquer outro meio de transferencia, segundo meus cálculos.
Mas sempre que a gente encontra um meio teoricamente mais barato, aparece uma notícia dizendo que teve aumento em algo.

_PS: Auditor fiscal que acessa os fórums do github: é mentira que fica mais barato viu?_ 😅

**Comment by samiraguiar** [comment]
@bergergit 1,1% é realmente o mais barato que vi. E tem outro ponto que pode ajudar também pra quem faz como pessoa física. [A partir de novembro](https://www.valor.com.br/financas/5866313/bc-cria-nova-alternativa-de-recebimento-de-remessas-por-pessoa-fisica) as remessas de até 10 mil reais de PF pra PF (seria da conta BB Américas pra conta daqui) podem ser feitas diretamente em reais, então creio que não terá mais IOF. Só tem que ver se eles não vão passar a cobrar as tarifas.

Se você seguir por esse caminho ou conseguir informações avisa aqui. Vou procurar saber se existe isso também pra Europa e se pra conta jurídica funciona diferente.

**Comment by bergergit** [comment]
@samiraguiar Lembrando que o IOF é cobrado em qualquer transferencia internacional mesmo (Payoneer, Bitwage, Banco Inter, etc). 

Agora essa notícia aí dos R$ 10.000,00 é interessantíssima hein?! 
Pelo que li, os bancos ainda vão ter que se regulamentar para adotar a medida. Mas certamente, penso que será muito mais facil e rápido o BB Americas se ajustar com o proprio Banco do Brasil em relaçao a isso. 

Por sinal, eu vou emitir NF pela minha empresa. Porem é uma empresa Individual, entao sei que nao tem problemas em relação a usar conta PF.
Possivelmente vou seguir esse caminho sim, ah nao ser que apareça alguma novidade nos próximos dias.
Vai ser um processo demorado, mas conforme for vendo novidades, posto aqui.

**Comment by yuriploc** [comment]
@bergergit como tem sido o uso do BB Americas? Estou pesquisando bastante nos fóruns e estou entre Payoneer, Husky, e agora BB Americas. Sou MEI e vou começar a prestar serviço para o exterior.

**Comment by fforbeck** [comment]
Olha, li que o IOF não deve ser cobrado quando a natureza da operação é Exportação de Serviço.
Alguns bancos cobram, mas não deveriam. Tem um [decreto](http://www.planalto.gov.br/ccivil_03/_Ato2011-2014/2014/Decreto/D8325.htm#art1) que isenta a alíquota do IOF para quem exporta bens e serviços. Atualmente eu não pago IOF no Inter.

**Comment by bergergit** [comment]
@yuriploc Eu ainda nao estou usando. Tive uns atrasos para começar as atividades para o exterior, então se for acontecer, será apenas la pela metade de 2019.
Mas pelas minhas pesquisas, BB Americas ainda seria a forma de receber dólares com menos taxas do que as outras opções. Se fosse começar o trabalho hoje, eu ainda seguiria com essa opção  (mas tem o inconveniente de ter que ter U$ 10k pra abrir conta remotamente).

@fforbeck Interessante isso hein. É aquele mesmo decreto sobre possibilidade de transferir até R$ 10.000,00 sem nenhuma taxa?

**Comment by samiraguiar** [comment]
@bergergit não, são diferentes.
Quanto ao IOF, quando vc recebe dinheiro pela Payoneer não é cobrado nada não, o dinheiro já cai em real. O que é cobrado é o spread deles de 2% em cima da cotação mid-market durante a transferência.

**Comment by JVMA8** [comment]
Alguém conhece a Hyperwallet? Sabe como funciona recebimento através deles?
